kitsune/users/migrations/0005_set_initial_contrib_email_flag.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""
+Set the `first_{revision,question}_email_sent` flags for existing users, to
+avoid back filling welcome emails to contributors.
+"""
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations
+from django.db.models import F
+
+
+def contrib_email_flags_forwards(apps, schema_editor):
+    Profile = apps.get_model('users', 'Profile')
+    Answer = apps.get_model('questions', 'Answer')
+    Revision = apps.get_model('wiki', 'Revision')
+
+    l10n_contributor_ids = set(
+        Revision.objects
+        .exclude(document__locale='en-US')
+        .values_list('creator', flat=True))
+    (Profile.objects
+        .filter(user__id__in=l10n_contributor_ids)
+        .update(first_l10n_email_sent=True))
```

Comment:
When I do data migrations, I like to see how many things are affected. So I do code like this:

https://github.com/mozilla/fjord/blob/83079bffb32d635375d1500af8cbe442cce04a34/fjord/feedback/south_migrations/0037_backfill_version.py#L53

That's an old south migration, but the gist of it is the same.

That way I see it locally, I see it in the deploy logs when I push it out and I don't see it in tests.